UPGRADE-4.0.md

```diff
@@ -228,6 +228,8 @@ Form
    ));
    ```
 
+ * Removed `ChoiceLoaderInterface` implementation in `TimezoneType`.
```

Comment:
I like both. Let's do both? 😄 

* Removed ``ChoiceLoaderInterface`` implementation in ``TimezoneType``. Use the "choice_loader" option instead:

  Before:
  
  ```php
  class MyTimezoneType extends TimezoneType
  {
      public function loadChoices()
      {
          // override the method
      }
  }
  ```
  
  After:
  
  ```php
  
  class MyTimezoneType extends AbstractType
  {
      public function. getParent()
      {
          return TimezoneType::class;
      }
  
      public function configureOptions(OptionsResolver $resolver)
      {
          $resolver->setDefault('choice_loader', ...); // override the option instead)
      }
  }
  ```